example/bwmf/shard_io_test.go

```diff
@@ -11,21 +11,15 @@ import (
 func TestShardIO(t *testing.T) {
 	client := filesystem.NewLocalFSClient()
 	path := "./.testShardIO.text.dat"
+	m, n := uint32(2), uint32(3)
 
 	oldShard := &pb.MatrixShard{
-		Row: []*pb.MatrixShard_RowData{
-			&pb.MatrixShard_RowData{At: make(map[int32]float32)},
-			&pb.MatrixShard_RowData{At: make(map[int32]float32)},
-		},
+		IsSparse: false,
+		M: m,
+		N: n,
+		Val: []float32 {0.70, 1.00, 0.90, 0.70, 0.80, 0.90},
 	}
 
-	oldShard.Row[0].At[0] = 0.70
-	oldShard.Row[0].At[1] = 1.00
-	oldShard.Row[0].At[2] = 0.90
-	oldShard.Row[1].At[1] = 0.70
-	oldShard.Row[1].At[2] = 0.80
-	oldShard.Row[1].At[3] = 0.90
-
 	client.Remove(path)
 	sErr := SaveMatrixShard(client, oldShard, path)
```

Comment:
This line seems to generate the file "./.testShardIO.text.dat" every time I run test script.
Can you delete it or add it to gitignore?

@5kg @BaiGang